xAPI.md

```diff
@@ -2639,34 +2658,48 @@ such a mechanism.
 (details not in this specification).
 * If this form of authentication is used to record Statements and no authority is specified, the LRS SHOULD 
 record the authority as a group consisting of an Agent representing the registered application, and an Agent 
 representing the known user.
 
 ###### Application registered + user unknown Process and Requirements
+**Process:** 
+The LRS honors requests that are signed using OAuth with the registered application's credentials and with an empty token and token secret.
 
-* The LRS honors requests that are signed using OAuth with the registered application's credentials and with an empty token and token secret.
-* If this form of authentication is used  to record Statements and no authority is specified, the LRS SHOULD 
+**Requirements:**
+* If this form of authentication is used to record Statements, the LRS SHOULD 
 record the authority as the Agent representing the registered application.
 
 ###### Application not registered + known user Process and Requirements
+**Process:**
+The AP uses a consumer secret consisting of an empty string to call the 
+Temporary Credential Request endpoint specifying the "consumer_name" and other usual parameters.
+The "consumer_name" contains a string representing the application requesting access. 
+
+The AP then sends the user's browser to the Resource Owner Authorization using the 
+temporary credentials obtained from the LRS. The Resource Owner Authorization presents a 
+page displaying the "consumer_name" plus a warning that the identity of the application requesting 
+authorization cannot be verified.
 
-* The AP MUST use a consumer secret consisting of an empty string.
-* Call "Temporary Credential" request.
-* Specify "consumer_name" and other usual parameters; User will then see "consumer_name" plus a warning 
-that the identity of the application requesting authorization cannot be verified.
-* The LRS MUST record an  authority that includes both that application and the authenticating user, as a group, 
-since OAuth specifies an application.
+Otherwise the process follows the standard OAuth workflow. 
+
+**Requirements:**
+* If this form of authentication is used to record Statements, the LRS SHOULD record an authority 
+that includes both that application and the authenticating user, as a group, since OAuth specifies 
+an application.
 
 ###### No application + known user Process and Requirements
+**Process:**
+Use a username/password combination provided by the LRS for use by the known user.
 
-* Use username/password combination that corresponds to an LRS login.
-* Authority to be recorded as the Agent identified by the login, **unless…**
-	* other Authority is specified **and…**
-	* LRS trusts the known user to specify this Authority.
+**Requirements:**
+* If this form of authentication is used to record Statements, the LRS SHOULD 
+record the authority as the Agent representing the known user.
```

Comment:
Completely re-worded this to match the style of the two similar requirements above. There's no need to specify that the LRS is allowed to keep the authority specified in the statement if it trusts the authority because that is the default behaviour.